demoapp/src/androidTest/java/me/uport/sdk/demoapp/CreateAccountTest.kt

```diff
@@ -2,25 +2,29 @@ package me.uport.sdk.demoapp
 
 import android.support.test.espresso.Espresso.onView
 import android.support.test.espresso.assertion.ViewAssertions.matches
+import android.support.test.espresso.matcher.ViewMatchers.isDisplayed
 import android.support.test.espresso.matcher.ViewMatchers.withId
 import android.support.test.espresso.matcher.ViewMatchers.withText
 import android.support.test.rule.ActivityTestRule
+import org.hamcrest.Matchers.allOf
 import org.hamcrest.Matchers.not
 import org.hamcrest.core.StringContains.containsString
 import org.junit.Rule
 import org.junit.Test
+import ro.mirceanistor.testutil.ViewMatcherIdlingRule
 
 class CreateAccountTest {
 
     @get:Rule
     val activityRule = ActivityTestRule(CreateAccountActivity::class.java)
 
+    @get:Rule
+    val viewMatcherIdlingRule = ViewMatcherIdlingRule(allOf(withId(R.id.progress), isDisplayed()))
```

Comment:
viewMatcherIdlingRule doesn't seem to ever be used. Is it necessary to have a variable for this?

Reply:
Yes, just as the activity rule, it is being used by the framework because of the annotation `@Rule`

It is the simplest way of implementing the idling resource without needing to polute the tests